packages/app/universal-ts-utils/src/array/sort.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Sorts an array of primitive types in either ascending or descending order.
```

Comment:
Would be nice to mention that we return a sorted copy, current description implies that we sort in place.

Reply:
Good point! I will update doc 🙏